spec/requests/objects_controller_validate_moab_spec.rb

```diff
@@ -9,7 +9,7 @@
   describe 'GET #validate_moab' do
     context 'when valid druid passed in' do
       before do
-        allow(ValidateMoabJob).to receive(:perform_later).and_return(true)
+        allow(ValidateMoabJob).to receive(:perform_later).and_return(ValidateMoabJob.new)
```

Comment:
tweaked to give slightly more realistic mocking since i was revising related code and adding tests anyway